File: SEMANA_5/FA-16185/SE5-PY/ejer4.py
cant = int(input("Ingrese la cantidad de numeros :"))
ceros = 0
pares = 0
impares = 0
for i in range(1, cant + 1):
    n = int(input(f"Ingresar numero {i} :"))

    if n == 0:
        ceros += 1
        #ceros = ceros +  i
    elif n % 2 == 0:
        pares += 1
    else:
        impares += 1

print("\nRESULTADOS")
print(f"Cantidad de pares : {pares}")
print(f"Cantidad de impares : {impares}")
print(f"Cantidad de ceros : {ceros}")

File: SEMANA_5/FA-16185/SE5-C/ejer4.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SE5_C
{
    internal class ejer4
    {   static void Main(string[] args)
        {
            int n;
            int ceros = 0, pares = 0, impares = 0;
            Console.WriteLine("Ingrese la cantidad de numeros :");
            int cant = int.Parse(Console.ReadLine());


            for (int i = 0; i < cant; i++)
            {
                Console.Write($"Ingresar numero {i+1} : ");
                n = int.Parse(Console.ReadLine());

                if (n == 0)
                {
                    ceros++;
                }
                else if (n % 2 == 0)
                {
                    pares++;
                }

                else
                {
                    impares++;
                }

            }
            Console.WriteLine("\n\nRESULTADOS");
            Console.WriteLine($"Cantidad de pares : {pares}");
            Console.WriteLine($"Cantidad de impares : {impares}");
            Console.WriteLine($"Cantidad de ceros : {ceros}");

        }
    }
}
